LU/siaburcharpat_con_culaind.trig

```diff
@@ -0,0 +1,129 @@
+@base <http://example.com/LU/siaburcharpat_con_culaind.trig>.
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix foaf:  <http://xmlns.com/foaf/0.1/> .
+@prefix rel: <http://purl.org/vocab/relationship/>.
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix dctype: <http://purl.org/dc/dcmitype/> .
+@prefix irishRel: <http://example.com/earlyIrishRelationship.ttl#> .
+@prefix prov: <http://www.w3.org/ns/prov#> .
+
+<http://example.com/LU> {
+<>
+    a dctype:Dataset;
+    dcterms:title "Siaburcharpat Con Culaind"@sga;
+    dcterms:isFormatOf <http://www.ucc.ie/celt/published/G301900/text025.html>;
+    dcterms:format "text/turtle" ;
```

Comment:
This needs to be `"application/trig"`.

Reply:
So, `dcterms:format "text/turtle" ;` to `dcterms:format "application/trig" ;`?

Reply:
Yep!

Reply:
Ok. There are a lot of other files that still have `dcterms:format "text/turtle" ;` . Is that ok?